### DISCUSSION TITLE
3.7.2 使用问题

### DISCUSSION THREAD

**Discussion by ttkkxfkl** [alibaba/arthas#2790] [Q&A]
3.7.2 直接使用java -jar 启动 提示 Logging system failed to initialize using configuration from 'classpath:log4j.xml'； 在源码中启动后访问8080进不了web ui ；  然后尝试了下  3.5.5 旧版本是可以的

**Comment by hengyunabc** [comment]
贴出具体的出错信息栈， 和来源。 arthas 本身是打包了 logback，不太可能出现加载 log4j。
